Assets/Scripts/FloatingOrigin/ThreeDTilesWorldTransformShifter.cs

```diff
@@ -0,0 +1,57 @@
+using System.Collections.Generic;
+using Netherlands3D.Coordinates;
+using Netherlands3D.Tiles3D;
+using UnityEngine;
+
+namespace Netherlands3D.Twin.FloatingOrigin
+{
+    public class ThreeDTilesWorldTransformShifter : WorldTransformShifter
+    {
+        private Dictionary<Transform, Coordinate> tilesToShift = new();
+
+        /// <summary>
+        /// Because the 3D Tile handler dynamically creates and destroys tiles, we need to collect a list of transforms
+        /// to reposition and calculate each their individual Coordinate, so that afetr shifting we can convert that
+        /// coordinate back to a Unity position. Because the calculation to a Unity position includes taking the origin
+        /// shift into account, it will thus reposition the tile to the correct location.
+        /// </summary>
+        public override void PrepareToShift(WorldTransform worldTransform, Coordinate from, Coordinate to)
+        {
+            tilesToShift.Clear();
+            
+            var contentComponents = transform.GetComponentsInChildren<Content>();
```

Comment:
Zijn er nog andere type objecten die hier als children kunnen bestaan? 
Anders zouden deze onafhankelijk kunnen maken van Content (en daarmee de 3DTiles package)

Dan zou dit bijv. een ChildrenWorldTransformShifter kunnen zijn, waar we simpelweg door de children van de Transform kunnen loopen.

Reply:
Die is er al, de ChildWorldTransformShifter, het probleem hier is dat bij de 3DTiles het specifiek de kinderen van de kinderen met het Content component zijn. Dus dit is 2 lagen diep en dat vond ik dan een soort van lastig om een ChildrenOfChildrenWorldTransformShifter te maken. Ik dacht: dan kan ik beter specifiek zijn ;)

Reply:
Zoals in Slack besproken, het is mooier om in de CartesianTiles en 3DTiles packages het instantieren van een Tile op basis van een prefab te doen en de WorldTransform op deze prefab te zetten en/of middels een prefabvariant dat in de twin te doen

Reply:
Dit is een goede vervolgactie